Client/Frontend/UIConstants.swift

```diff
@@ -60,6 +60,26 @@ private struct TempStrings {
     // Bug 1186013 - Prompt for going to clipboard URL
     let goToCopiedURL = NSLocalizedString("Go to copied URL?", comment: "Pending feature; currently unused string! Prompt message shown when browser is opened with URL on the clipboard")
     let goToCopiedURLButton = NSLocalizedString("Go", comment: "Pending feature; currently unused string! Button to browse to URL on the clipboard when browser is opened")
+
+    // Bug 1233418 - Login Manager Strings
+    let loginsListTitle                 = NSLocalizedString("Logins", tableName: "LoginManager", comment: "Title for Logins List View screen")
+    let loginSearchFieldTitle           = NSLocalizedString("Search", tableName: "LoginManager", comment: "Title for the search field at the top of the Logins list screen")
+    let clearSearchAccessibilityLabel   = NSLocalizedString("Clear Search", tableName: "LoginManager", comment: "Clears the search input field and exits out of input mode")
+    let searchOverlayAccessibilityLabel = NSLocalizedString("Enter Search Mode", tableName: "LoginManager", comment: "Accessibility label for entering search mode for logins")
+    let detailUsernameRowTitle          = NSLocalizedString("username", tableName: "LoginManager", comment: "Title for username row in Login Detail View")
+    let detailPasswordRowTitle          = NSLocalizedString("password", tableName: "LoginManager", comment: "Title for password row in Login Detail View")
+    let detailWebsiteRowTitle           = NSLocalizedString("website", tableName: "LoginManager", comment: "Title for website row in Login Detail View")
+    let deleteLoginDetail               = NSLocalizedString("Delete", tableName: "LoginManager", comment: "Button in login detail screen that deletes the current login")
+    let lastLoginModified               = NSLocalizedString("Last modified %@", tableName: "LoginManager", comment: "Footer label describing when the login was last modifed")
+    let revealPassword                  = NSLocalizedString("Reveal", tableName: "LoginManager", comment: "Reveal password text selection menu item")
+    let openAndFill                     = NSLocalizedString("Open & Fill", tableName: "LoginManager", comment: "Open and Fill website text selection menu item")
+    let deselectAll                     = NSLocalizedString("Deselect All", tableName: "LoginManager", comment: "Title for deselecting all selected logins")
+    let selectAll                       = NSLocalizedString("Select All", tableName: "LoginManager", comment: "Title for selecting all logins")
+    let deletePrompt                    = NSLocalizedString("Are you sure you want to delete all your logins?", tableName: "LoginManager", comment: "Prompt message warning user about deleting all of their logins")
```

Comment:
I would change these, given #1359 and the strings we sketched out in Bug 1225224, to

```
    Are you sure?
    This will permanently delete your saved logins.
    [Cancel]       [Delete]
```

or, if Sync is attached and syncing passwords:


```
    Are you sure?
    All logins will be deleted from your connected devices.
    [Cancel]       [Delete]
```

and similar for the selected logins case ("Selected logins will be deleted from your connected devices.").

@tecgirl can opine more.